crates/bevy_gizmos/src/lib.rs

```diff
@@ -315,6 +341,8 @@ struct LineGizmo {
     colors: Vec<LinearRgba>,
     /// Whether this gizmo's topology is a line-strip or line-list
     strip: bool,
+    /// Whether this gizmo should draw line joins. This is only applicable if the gizmo's topology is line-strip.
+    joins: GizmoLineJoint,
```

Comment:
This should be `joints` (and in the docs above), and `joins_resolutions`

Reply:
There were a couple of those but I think I've got them all now (hopefully).

I originally thought they were called line-joins 😅